website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -54,6 +54,10 @@ If you already use gRPC encryption, change the following fields to ensure compat
 
 + Change `ports.grpc` to `ports.grpc_tls`. Refer to the [`grpc_tls_port` documentation](/docs/agent/config/config-files#grpc_tls_port) for details.
 + Change `addresses.grpc` to `addresses.grpc_tls`. Refer to the [`grpc_tls` documentation](/docs/agent/config/config-files#grpc_tls) for details.
++ Set the gRPC CA certificate with one of the new configuration options:
```

Comment:
```suggestion
+ Update `consul connect envoy` command invocations to specify gRPC CA certificates with one of the new configuration options:
```

I think this is a little clearer what needs updating. Otherwise users might think they need to set this for the agent's themselves.